devices/maas2/maas2.py

```diff
@@ -301,8 +315,8 @@ def check_test_image_booted(self):
             "/bin/true",
         ]
         try:
-            subprocess.check_output(cmd, stderr=subprocess.STDOUT, timeout=60)
-        except Exception:
+            subprocess.check_call(cmd, stderr=subprocess.STDOUT, timeout=60)
```

Comment:
Any reason not to use run() here for consistency and to follow recommended approach?

Reply:
Not really. check_call() is really just run() with check=True. I changed it since this was the only usage of it. The main thing here is that I changed it from check_output since we don't care about any return from it, just whether or not it works.